# Properties of single loci

## Introduction

too easy 

## Allele and genotype frequencies

too easy

## The transmission of genetic information

### The Hardy-Weinberg principle

\begin{equation}
  p^2 + 2pq + q^2 = 1
  (\#eq:hardy-weinberg-principal)
\end{equation}

where $p$ = allele frequency of first allele at a locus and $q$ = allele frequency of the second allele at that same locus.

Assumptions of H-W:

* No selection
* No mutation
* Random mating
* No differential migration
* No random drift

Even though these assumptions will never be met completely in the real world, for the majority of the time the H-W prinicple holds regardless.

Assuming assumptions are met, 2 important points from H-W:

1. It takes no more than a single generation to equilibriate and stabilize gene frequencies in the two sexes. 
2. Only one additional generation is required for the stabilisation of the genotype frequencies into the predictible Hardy-Weinberg proportions.

### Sex-linked loci

Alleles on sex chromosomes in diploid organisms are obviously different. Sons can only receive an X chromosome from their mother so the frequency of X linked loci in the sons is equal to that of their mothers. Daughters receive both an X chromosome from Mum + from Dad. 

Overall this means allele frequencies oscilate around an equilibrium state, but continually get closer to that state over the generations (see Figure 4.2 and page 56 for equation).

### Polyploidy

Skipped over this section because it's not relevant to human quant gen. Buuut, essentially it just details how to derive allele frequencies under a certain case of polyploidy. Also, it should be noted that of course H-W does not hold under polyploidy!

### Age structure

Age structure also complicates our idealised model of H-W. In populations composed of several age classes, the generations overlap, and this causes the approach of genotype frequencies towards the H-W expectations to be gradual (rather than just by 1 or 2 generations), even in the case of an autosomal locus. Doesn't explain this very much, but it's covered elsewhere. Importantly, when newly founded populations have significant age structure, fluctuations in both gene and genotoype frequencies may occur for a substantial period of time even in the abscence of selection!

### Testing for Hardy-Weinberg proportions

Says in the book that LRT can be used to test for departures from HWE and it can, but another common method is the chi-squared test and in [PLINK](https://www.cog-genomics.org/plink/1.9/) they used Haldane's exact test which is apparently analogus to Fisher's exact test ([papier on it](https://www.genetics.org/content/183/4/1431)) (can also use Fisher's exact test if the sample size is tiny and the allele is rare.). Essentially, in a population, at a specific locus, you can calculate the allele frequencies (and from that expected genotype frequencies) from the observed genotype frequencies then test if there is a difference between the observed and expected values. LRT equation for it given on page 60. See code for some comparisons.

Should remember (as pointed out above), that just because some assumptions are violated, doesn't mean you'd get a departure from HWE!

## Characterising the influence of a locus on the phenotype

If a trait is entirely influenced by a single locus then the genetic effect on that trait can be characterised pretty easily and the dominance and additive effects of the alleles can be calculated. So if a locus has genotypes $B_1B_1$, $B_1B_2$, $B_2B_2$, then the values given to these genotypes can be said to be: $-a$, $(1 + k)a$ and $+a$. Now if you have genotype data at that locus and data on the trait you can work out the effect of the $B_2$ allele by taking the mean phenotypic value of individuals with $B_2B_2$ and subtracting the mean phenotypic value of individuals with $B_1B_1$ and dividing by 2 i.e.

\begin{equation}
  B_{2eff} = \frac{p_{B2} - p_{B1}} {2}
  (\#eq:effect-of-b2)
\end{equation}

where $B_{2eff}$ is the effect of allele $B_2$, $p_{B2}$ is the mean phenotypic value of individuals with $B_2B_2$ and $p_{B1}$ is the mean phneotypic value of individuals with $B_1B_1$.

As $B_{2eff} = a$ you can then substitute this in to $(1 + k)a$ to get the dominance coefficient $k$. Of course if $k = 0$ then there is no dominance (in reality you would calculate probability of dominance).

## The basis of dominance

Confusing part... Don't really get the enzyme activity bit...

Main point (I think) is that new deleterious mutations are very likely to be recessive and new mutations with a slight deleterious effect interact in an almost entirely additive fashion (no dominance!).

## Fisher's decomposition of the genotypic value

Recalling that the phenotypic value can be partitioned like so: 

\begin{equation}
  z = G + E
  (\#eq:phenotypic-value-partition)
\end{equation}

where $z$ is the phenotype, $G$ is the genotypic value and $E$ is the environmental value. 

The genotypic value of a specific locus can be partitioned into it's "expected" values based on there being only additive effects ($\hat{G}$) and the deviations from the expected values or dominance effects ($\delta$). So for genotype $B_iB_j$:

\begin{equation}
  G_{ij} = \hat{G_{ij}} + \delta_{ij}
  (\#eq:expected-genotypic-value-simple)
\end{equation}  

This can be formalised (whatever the fuck that means) by regressing the genotypic values on the number of $B_1$ and $B_2$ alleles in the genotype ($N_1$ and $N_2$):

\begin{equation}
  G_{ij} = \hat{G_{ij}} + \delta_{ij} = \mu_G + \alpha_1N_1 + \alpha_2N_2
  (\#eq:expected-genotypic-value-2)
\end{equation}

$\mu_G$ = the mean genotypic value in the population, $\alpha_1$ and $\alpha_2$ are the slopes of the regression, $N_1$ and $N_2$ are the number of $B_1$ and $B_2$ alleles. So the regression is:

\begin{equation}
  G_{ij} ~ N_2 + N_1
\end{equation}

By noting that for any individual, $N_1 = 2 - N_2$ you can reduce the multiple regression model into an easier to work with univariate model. Give it a go (use equation \@ref(eq:expected-genotypic-value-2)): 

\begin{equation}
\begin{split}
  G_{ij} &= \mu_G + \alpha_1(2 - N_2) + \alpha2N_2 + \delta_{ij} \\
  &= l + (\alpha_2 - \alpha_1)N_2 + \delta_{ij}
\end{split}
(\#eq:genotypic-value-regression)
\end{equation}

where $l = \mu_G + 2\alpha_1$ is the intercept and the slope is $\alpha = \alpha_2 - \alpha_1$

If you plotted the genotypic value ($G$) against gene content ($N_2$ or number of $B_2$ alleles) and calculated residuals these residuals would be $\delta$, the dominance deviation (see Figure 4.6).

The rest of the chapter uses this regression and what we know about genotype frequencies to derive a formula for the average effect of allelic substitution:

\begin{equation}
  \alpha = a[1 + k(p_1 - p_2)]
  (\#eq:allelic-sub-effect)
\end{equation}

where $a$ = genotypic value of $B_2$ (see above), $k$ is the dominance coefficient and $p_1$ and $p_2$ are the frequencies of $B_1$ and $B_2$. This value $\alpha$ represents the average change in genotypic value that results when a $B_2$ allele is randomly substituted for a $B_1$ allele. If no dominance ($k = 0$) then $\alpha = a$. Except in the case of additivity, the average effect of allelic substitution is not simply a function of the inherent physiological properties of the allele. It can only be defined in the context of the population!

## Partioning the genetic variance.

Deriving variance of $G$: 

\begin{align}
\begin{split}
  G &= \hat{G} + \delta \notag \\
  \sigma^2_G &= \sigma^2(\hat{G} + \delta) \notag \\
  &= \sigma^2(\hat{G}) + 2\sigma(\hat{G} + \delta) + \sigma^2(\delta)
\end{split}
(\#eq:total-genetic-variance-longwinded)
\end{align}

The top equation is just like a regression, with $\delta$ being the residual error and we know that for least-squares there is no correlation between the residual error and the predictor. So there is no correlation between $\hat{G}$ and $\delta$. Therefore:

$$ \sigma^2_G = \sigma^2(\hat{G}) + \sigma^2(\delta) $$
OR more commonly
\begin{equation}
  \sigma^2_G = \sigma^2_A + \sigma^2_D
  (\#eq:genetic-variance-one-locus)
\end{equation}

$\sigma^2_A$ is the variance of $G$ explained by regression on $N_2$ (or $N_1$), and $\sigma^2_D$ is the residual variance of that regression. The variance of the additive and dominance effects!

For a diallelic locus we can do some rearranging of equations in Table 4.1 of the book and get these equations:

\begin{align}
  \sigma^2_A &= 2p_1p_2\alpha^2 (\#eq:additive-var) \\
  \sigma^2_D &= (2p_1p_2ak)^2 (\#eq:dominance-var)
\end{align}

From these we can clearly see that both components depend on allele frequencies, the dominance coefficient and the homozygous effect (remember $\alpha$ is just the slope of the $G ~ N_2$ regression!).

By plotting how genetic variance changes with gene frequency under different scenarios (see \@ref(fig:sigma-g-gene-frequency)). You see some interesting patterns. Firstly, at a single diallelic locus, you see that $\sigma^2_A$ reaches it's peak when $p_1 = p_2 = 0.5$. Secondly, it's clear that, even in the case of overdominance (which is rare!), additive genetic variance will almost always be much higher than genetic variance from dominance effects, even when the frequency of the dominant allele is high.

(ref:sigma-g-gene-frequency-cap) The dependence of components of genetic variance at a locus on the frequency of the $B_2$ allele. $a$ is set to be one, which scales the vertical axes so that for any particular case, the actual variances are obtainable by multiplying by $a^2$.

```{r sigma-g-gene-frequency, message=FALSE, warning=FALSE, echo=FALSE, results="asis", fig.cap='(ref:sigma-g-gene-frequency-cap)'}
	library(tidyverse)
	library(gridExtra)
	p2 <- seq(0, 1, length.out = 101) # allele frequency of allele 2
	p1 <- 1 - p2 # allele frequency of allele 1
	k <- c(0, -1, 1, 2) # dominance effect
	a <- 1 # additive effect
	calc_alpha <- function(p1, p2, a, k) a * (1 + k * (p1 - p2))
	calc_sigmaA <- function(p1, p2, a, k) {2 * p1 * p2 * calc_alpha(p1, p2, a, k)^2
	}
	calc_sigmaD <- function(p1, p2, a, k) (2 * p1 * p2 * a * k)^2
	calc_sigmaG <- function(sigmaA, sigmaD) sigmaA + sigmaD
	plots <- lapply(k, function(kval) {
		sigmaA <- calc_sigmaA(p1, p2, a, kval)
		sigmaD <- calc_sigmaD(p1, p2, a, kval)
		sigmaG <- calc_sigmaG(sigmaA, sigmaD)
		tab <- tibble(p2 = p2, A = sigmaA, D = sigmaD, G = sigmaG) %>%
			pivot_longer(-p2, names_to = "variance_component", values_to = "genetic_variance")
		p <- ggplot(tab, aes(x = p2, y = genetic_variance, colour = variance_component)) +
			geom_line() +
			labs(x = "p2, gene frequency", y = "Genetic variance", colour = NULL, title = paste("k =", kval)) +
			theme_bw()
		return(p)
	})
	marrangeGrob(plots, nrow=2, ncol=2, top = NULL)
```

## Additive effects, average excesses and breeding values

The dominance deviation of a parent, which is a function of the interaction between the two parental alleles, is eliminated when gametes are produced. Thus, one can think of $\hat{G}$ and $\delta$ as the heritable and nonheritable components of an individual's genotypic value.

Fisher proposed two different measures of the effect of an allele: one being the additive effect ($\alpha_i$) and then the average excess $\alpha^x_i$. The average excess $\alpha^x_2$ of allele $B_2$ is the difference between the mean genotypic value of individuals carrying at least one copy of $B_2$ and the mean genotypic value of a random individual from the entire population:

\begin{equation}
  \alpha^x_2 = (G_{12}P_{12|2} + G_{22}P_{22|2}) - \mu_G
  (\#eq:average-excess-effect)  
\end{equation}

where $P_{ij}$ is the conditional probability of a $B_iB_j$ genotype given that one allele is $B_i$. Under random mating $P_{ij|i} = p_j$ ($p_j$ = frequency of allele $B_j$). THINK ABOUT HARDY-WEINBERG AND IT MAKES SENSE!

So under random mating,

\begin{equation}
  \alpha^x_2 = G_{12}p_1 + G_{22}p_2 - \mu_G
  (\#eq:average-excess-effect2)
\end{equation}

$G_{12} = a(1+k)$ and $G_{22} = 2a$. By substituting these into the equation above for $\alpha^x_1$ and $\alpha^x_2$ and then calculating $\alpha_1$ and $\alpha_2$ (additive effects) by the method previously mentioned (regressing genotypic value $G$ on the number of $B_2$ alleles, $N_2$), we will see they're equivalent (shown on page 72): 

\begin{align}
  \alpha_2 &= p_1\alpha \notag \\
  \alpha_1 &= -p_2\alpha
\end{align}

The breeding value of an individual ($A$) is the sum of the additive effects of its genes. So the breeding value of a $B_1B_1$ homozygote is just $2\alpha_1$. In randomly mating populations the breeding value of a genotype is equivalent to twice the expected deviation of its offspring mean phenotype from the population mean. Soooo, no genotype information is needed to calculate the breeding value. All we have to do is mate an individual to many randomly chosen individuals from the population and taking twice the deviation of its offspring mean from the population mean. EASY IN HUMANS!!!

In Chapter 13 this will be discussed wrt candidate gene studies.

## Extensions for multiple alleles and non random mating

So this section seems mostly unrelevant as we're unlikely to deal with situations with more than 2 alleles. Non-random mating could be encountered if we're interested in some phenotypes (e.g. alcohol intake). Buuuut, it's still good to note some of the generalised equations for what we've been discussing so far in the chapter.

### Average excess

When $n$ alleles are present, the average excess, $\alpha^x_i$, for any allele $B_i$ is given by

\begin{equation}
  \alpha^x_i = \sum_{j=1}^{n} P_{ij|i}G_{ij} - \mu_G
  (\#eq:average-excess-effect3)
\end{equation}

Remember, under random mating $P_{ij|i} == p_j$

### Additive effects

The genotypic value can also be obtained using regression as before, but in it's generalised form is a multivariate regression. For $n$ alleles 

\begin{equation}
  G = \mu_G + \sum_{i=1}^{n} \alpha_{i}N_{i} + \delta
  (\#eq:generalised-genotypic-value-regression)
\end{equation}

After some re-arranging can derive the regression coefficients and finally end with

\begin{equation}
  \alpha_i = \sum_{j=1}^{n} p_jG_{ij} - \mu_G
  (\#eq:all-average-effects)
\end{equation}

i.e. under random mating, the average effects ($\alpha_i$) are equal to the conditional mean deviations from the mean genotypic value of the population ($\mu_G$).

For non-random mating we need the inbreeding coefficient, $f$ to define our genotype frequencies:

\begin{align}
  P_{ii} &= (1 - f)p^2_i + fp_i \notag \\
  P_{ij} &= 2(1 -f)p_ip_j
  (\#eq:nonrandom-mating-genotype-frequencies)
\end{align}

Unsure of why, but this means

\begin{equation}
  \alpha_i = \frac{\alpha^x_i} {1 + f}
  (\#eq:average-effects-inbreeding)
\end{equation}

so $f$ is the fractional reduction of heterozygote frequencies relative to those expected under random mating. This means you can kind of do a test for random mating by checking heterozygote and homozygote frequencies in a population!

### Additive genetic variance

The additive genetic variance across $n$ alleles is

\begin{equation}
  \sigma^2_A = 2 \sum_{i=1}^{n} p_i\alpha_i\alpha^x_i
  (\#eq:all-additive-genetic-variance)  
\end{equation}

In general inbreeding inflates the additive genetic variance by causing correlations among the effects of alleles within the same individuals.

The broad sense heritability, even under scenarios of non-random mating can be given by

\begin{equation}
  \sigma^2_G = \sigma^{2}(\alpha_i + \alpha_j) + \sigma^{2}(\delta_{ij})
\end{equation}

although it should be noted that the definitions of $\alpha_i$ and $\delta_ij$ change with the degree of inbreeding! Random mating means $\alpha_i$ and $\alpha_j$ are uncorrelated so we get back to the good old equation

$$ \sigma^2_G = \sigma^2_A + \sigma^2_D $$

Importantly, under random mating, $\sigma^2_A$ is equivalent to the variance of breeding values of individuals in the population.

***
**Summarising some key terms**

__The homozygous effect, $a$, and the dominance coefficient, $k$,__ are intrinsic properties of allelic products. They are not functions of allele frequencies, but may vary with genetic background

__The additive effect, $\alpha_i$, and the average excess, $\alpha^x_i$,__ are properties of alleles in a particular population. They are functions of $a$, $k$ and genotype frequencies ($p_i$).

__The breeding value, $A$,__ is a property of a particular individual in reference to a particular population. It's equivalent to the sum of the additive effects of an individual's alleles.

__The additive genetic variance, $\sigma^2_A$__ is a property of a particular population. It is equivalent to the variance of the breeding values of individuals within the population.

***

## End of chapter questions

1. What is the Hardy-Weinberg principle and what are it's assumptions?
2. What does the H-W principle mean for gene and genotype frequencies across generations?
3. What is age structure and how does it affect HWE?
4. How can you test for HWE?
5. Are deliterious mutations likely to be dominant or recessive?
6. Assuming a trait was entirely influenced by a single locus, how could you calculate dominance and additive effects knowing the genotypes and phenotypes of the individuals in the sample?
7. What is the formula for the average effect of allelic substitution?
8. For a diallelic locus, what does the additive genetic variance and dominance genetic variance depend on?
9. How does the contribution of additive genetic variance to total genetic variance change when $k$ varies?
10. What is the breeding value of an individual?
11. Define the additive genetic variance in the presence of $n$ alleles
12. Learn the definitions of the key terms!


