# Sources of genetic variation for multilocus traits

## Epistasis

Epistasis describes the nonadditivity of effects between loci, i.e. the alleles of one loci influence the effects of another loci. 

The genotypic value, $G_{ijkl}$, needs to take into account all the interaction terms that can arrive between loci, for two loci it's additive x additive effects ($\alpha\alpha$), additive x dominance effects ($\alpha\delta$), and dominance x dominance effects ($\delta\delta$). As the number of loci increases the number of interaction terms increase steadily e.g. $\alpha\alpha\alpha$ will be there for three loci.

## A general least-squares model for genetic effects

This is just an extension of the one-locus linear model introduced in Chapter 4. 

For this section, imagine we are interested in measuring the genetic effects of two loci, $G_{ijkl}$, which can easily be extended to more. The additive effect of an allele on a phenotype is just the phenotypic value in people with that allele minus the mean phenotypic value of the population. When considering epistatic effects we can define it in the same way. 

\begin{equation}
  \alpha_{i} = G_{i...} - \mu_{G}
  (\#eq:additive-effects-i)
\end{equation}

$G_{i...}$ is just the conditional mean phenotype of individuals with allele $i$ at the first locus without regard to the other allele at that locus or to the genotype at the second locus. The other additive terms (for $\alpha_{j}$, $\alpha_{k}$, $\alpha_{l}$) are defined in the same way. Within each locus, the mean value of average effects (weighted by allele frequency) = 0. 

Dominance effects can be defined in a similar way, complete these equations by recalling \@ref(eq:generalised-genotypic-value-regression): 

\begin{align}
  \delta_{ij} &= G_{ij..} - ... (\#eq:dom-effects-ij) \\
  \delta_{lk} &= G_{..lk} - ... (\#eq:dom-effects-lk)
\end{align}

Like with the additive effects, the mean dominance deviation at each locus is equal to zero.

Epistatic effect terms proceed in a similar fashion. Letting $G_{i.k.}$ be the mean phenotype of individuals with gene $i$ at locus 1 and $k$ at locus 2, without regard to the other two genes, the $ik$th additive x additive effect is:

\begin{equation}
  \left(\alpha\alpha\right)_{ik} = G_{i.k.} - \mu_{G} - \alpha_{i} - \alpha_{k} 
  (\#eq:add-add-effects-ik)
\end{equation}

So $\left(\alpha\alpha\right)_{ik}$ is the deviation of the conditional mean $G_{i.k.}$ from the expectation based on the population mean $\mu_{G}$ and the additive effects $\alpha_{i}$ and $\alpha_{k}$. An additive x dominance effect measures the interaction between an allele at one locus with a genotype of another locus (see equation 5.5 in book) and the dominance x dominance effect involves an interaction between the genotypes at each locus (see equation 5.6 in book). 

\begin{equation}
  TO DO
  (\#eq:complete-genotypic-value)
\end{equation}

The complete genotypic value, $G_{ijkl...}$ can be found in equation 5.7 in the book. These parameters depend on genotype frequencies in the population, but the mean value of each type of effect is always equal to zero.

The genotypic value of an individual is often impossible to quantify because of variation in the phenotype due to the environment, but the genotypic value for an individual equation can be extended to populations. Providing mating is random and segregation of loci is independent, there is no statistical relationship between the genes found within or among loci. So the total genetic variance is just the sum of the variance of the individual effects, simplified this is:

\begin{equation}
  \sigma^{2}_{G} = \sigma^{2}_{A} + \sigma^{2}_{D} + \sigma^{2}_{AA} + \sigma^{2}_{AD} + \sigma^{2}_{DD} + ...
  (\#eq:multilocus-genetic-variance)
\end{equation}

... here and in other cases just symbolises more terms can be added if more than two loci are used.

Epistatic effects are expected to be common throughout the genome and Wright thought they were the rule, rather than exception. See example two in the book for calculations of epistatic effects and how much variance they contribute to the overall genetic variance component. Overall, it is clear that even with large epistatic effects, additive genetic variance, $\sigma^{2}_{A}$ will pretty much always (if not always) contribute to the vast majority of overall genetic variance $\sigma^{2}_{G}$. This is important for two reasons: 

1. Variance components provide limited insight into the physiological mode of gene action, i.e. just because genetic variance is explained by additive effects (which means you essentially count each gene separately), it does not mean the interaction between genes is not important in terms of their function!
2. When interested in the variance of a trait that is explained by genetics, you can expect the vast majority of that variance to be explained by additive genetic effects, which makes things like estimating heritability far easier.

### Extension to haploids and polyploids

Skipped this section as not relevant to humans.

## Linkage

Genes of the same chromosome tend to be inherited as a group, a tendency that declines with increasing distance between the loci. Crossing-over during meiosis is responsible for this decline. 

***
**Difference between linkage and linkage disequilibrium**

Loci are linked if they tend to be inherited together. If loci are correlated for any reason (don't need to be inherited together), they are in linkage disequilibrium. The census units for measuring linkage are gamete frequencies, so you can use an individual to estimate this. LD is measured across a population.

Can get linkage without LD, can get linkage and LD, and can get two correlated loci (LD) that aren't linked.

***

Even though you can get correlated loci for reasons other than linkage, the LD between linked loci are more likely to persist over time as seen in \@ref(eq:LD-decay).

Under linkage equilibrium, the frequency of gametes is the product of allele frequencies, so for loci $A$ and $B$,

\begin{equation}
  Freq(AB) = Freq(A) * Freq(B)
  (\#eq:linkage-equilibrium)
\end{equation}

So A and B are independent of each other.

Measure of disequilibrium is just the departure from this:

\begin{equation}
  D_{AB} = Freq(AB) - Freq(A) * Freq(B)
  (\#eq:LD-D)
\end{equation}

$D_{AB}$ can be  positive or negative depending on whether $A$ and $B$ are in coupling ($AB$ gametes are overrepresented) or repulsion ($AB$ gametes are underrepresented) disequilibrium.  $D$ is often referred to as the coefficient of linkage disequilibrium (although can be non-zero without linkage!).

Selection, migration, mutation and drift can help maintain LD. Even without these forces, once LD is established it can be maintained for many generations (especially if loci are more tightly linked!).

Expected LD changes over time depend on the recombination fraction between loci, $c$. This value ranges from 0 to 0.5, where 0 essentially means the loci are inherited together and 0.5 is free recombination between loci.

If recombination frequency between the $A$ and $B$ loci is $c$, the disequilibrium in generation $t$ is given by:

\begin{equation}
  D(t) = (1 - c)^{t}D(0)
  (\#eq:LD-decay)
\end{equation}

This equation is graphically displayed in \@ref(fig:ld-decay).

(ref:ld-decay-cap) The decline, under random mating, of linkage disequilibrium when the initial value [D(0)] is set to 1 as a function of the recombination frequency, c.

```{r ld-decay, message=FALSE, echo=FALSE, warning=FALSE, results="asis", fig.cap='(ref:ld-decay-cap)'}
	library(tidyverse)
	calc_D <- function(D0=1, c, t) {
		D <- (1 - c)^t * D0
		D
	}
	c <- c(0.001, 0.01, 0.1, 0.5)
	t <- 0:100
	D_df <- expand.grid(c = c, t = t) %>%
		mutate(D = calc_D(1, c, t)) %>%
		arrange(c, t)
	p <- ggplot(D_df, aes(x = t, y = D, colour = as.factor(c))) +
		geom_line() +
		theme_bw() +
		labs(x = "Generation (t)", y = "D", colour = "c")
	print(p)
```

To estimate $D$ you can directly count gamete frequencies of an individual. This is not possible for most organisms though and usually you have to make-do with measured multilocus genotypes across a population. You can then workout gametes used to produce the genotypes (e.g. someone with an $AABb$ genotype would be formed from a $AB$ gamete and $aB$ gamete). However, with double heterozygotes you can't be sure if an $ABab$ individual was formed from $AB$ and $ab$ gametes or from $Ab$ and $aB$ gametes. Under random mating, it's not necessary to distinguish between coupling and repulsion heterozygotes so this doesn't really matter. In this case an estimate of $D$ is given by:

\begin{equation}
  \hat{D_{AB}} = \frac{N} {N - 1} [\frac{4N_{AABB} + 2(N_{AABb} + N_{AaBB}) + N_{AbBb}} {2N} -2\hat{p_{A}}\hat{p_{B}}]
  (\#eq:LD-from-genotypes)
\end{equation}

The equation for the sampling variance of $D$ is shown on page 99.

Ideally you'd have 1000s of samples to achieve reasonable statistical power when estimating $D$ using multilocus genotype frequencies.

## Effect of disequilibrium of the genetic variance

The aggregate effects of gametic phase disequilibrium might be extensive for quantitative traits whose expression is based on large numbers of loci, even if the average level of disequilibrium between pairs of loci is relatively small. If genes with a positive influence on a character tend to be associated on some chromosomes, and those with a negative influence on others (coupling disequilbrium), the observed genetic variation will be inflated relative to the expectation under random assortment. The opposite will occur if "plus" alleles at one locus tend to be associated with "minus" alleles at another (repulsion disequilibrium). This is illustrated nicely by figure 5.6 in the book. Think of it this way, you're studying gene $A$, gene $B$ and phenotype $X$. If upregulation of $A$ leads to an increase in $X$ and an upregulation of $B$ leads to an increase in $X$ by the same proportions, then if genetic variation always occurs so that whenever $A$ is upregulated, $B$ is downregulated by the same amount, then what will be observed at both of the loci is that variation at them is not associated with variation in $X$. This situation described is complete repulsion linkage (again, see figure 5.6 in book). This is assuming additive effects across loci. When there is no disequilibrium between loci, the variance at each locus is just $2pqa^{2}$, see \@ref(eq:additive-var).

As always, dominance effects muddy the waters, but here are the formalized multilocus analogs of \@ref(eq:additive-var) and \@ref(eq:dominance-var):

\begin{align}
  \sigma^2_A &= 2\sum_{i=1}^{n} \alpha(i)^{2}p_{i}q_{i} + 2\sum_{i=1}^{n} \sum_{j\ne1}^{n} \alpha(i)\alpha(j)D_{ij} (\#eq:additve-var-LD) \\ 
  \sigma^2_D &= 4\sum_{i=1}^{n} (a_{i}k_{i}p_{i}q_{i})^{2} + 4\sum_{i=1}^{n} \sum_{j\ne1}^{n} a_{i}a_{j}k_{i}k_{j}D^2_{ij} (\#eq:dominance-var-LD)
\end{align}

where $\alpha(i)$ is the average effect of allelic substitution at   the $i$th locus (defined in equation \@ref(eq:allelic-sub-effect)).

Epistatic interactions make things crazy complicated!

In summary, the componenets of expressed genetic variance for quantitative traits can be partitioned into expected values under gametic phase equilibrium and deviations from these caused by disequilibrium. From the book: "When the disequilibrium covariance is negative, we refer to it as hidden genetic variance because it is subject to conversion to expressed genetic variance via the breakdown of gametic phase disequilibria". What I think this means: Negative disequilibrium covariance is just when correlated loci covary in a way so that when one increases a traits value, the other decreases it, this is what happens in repulsion linkage! The reason it's not just called repulsion linkage is because you can have covarying loci without them being linked. Sooo, what the book is saying here, is that if linkage disequilibrium (correlation) between the two loci that negatively covary is reduced, the amount of genetic variance they explain in the trait will increase! 

### The evidence

Hidden genetic variation is expected to be a natural consequence of stabilising selection, which favours linkage groups for their composite properties without regard to the alleles at individual loci. Theoretical work has suggested that stabilising selection encourages the development of substantial hidden genetic variance, potentially depressing the level of expressed genetic variance to 50% or less than its equilibrium expectation.

Of course selection doesn't always favour an increase in hidden genetic variance. Sometimes coupling selection is favoured, so that expressed genetic variance exceeds equilibrium expectations. In this case the disequilibrium covariance is positive, and recombination would be expected to result in a reduction in the expressed genetic variance.

Here we've just considered one trait, but of course the same thinking applies to selection upon multiple traits simultaneously. For example, in populations of insects (LIKE BEES) that exploit multiple host plants, one might expect a genetic correlation to evolve such that individuals prefer to feed on the plant species upon which they perform best. Such correlations could result from LD between a set of genes influencing preference and another influencing performance.

## End of chapter questions

1. Define epistasis
2. Describe the terms that will be needed to define the genotypic value $G_{ijkl}$
3. Complete equations \@ref(eq:dom-effects-ij) and \@ref(eq:dom-effects-lk)
4. Give two important inferences from the fact the total genetic variance will mostly be attributable to the additive genetic variance, even if there are large dominance and epistatic effects
5. What is linkage and linkage disequilibrium? 
6. What is the coefficient of linkage disequilibrium?
7. What can influence maintenance of LD? (5 things)
8. What is the relationship between linkage and LD over time?
9. Give two methods of estimating $D$
10. What are coupling disequlibrium and repulsion disequilibrium and how do they effect genetic variance?
11. Explain equations \@ref(eq:additve-var-LD) and \@ref(eq:dominance-var-LD)
12. What is hidden genetic variance?
13. How is stabilising selection thought to influence hidden genetic variance? Why?

