<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Covariance, regression, and correlation | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits</title>
  <meta name="description" content="Chapter 3 Covariance, regression, and correlation | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  <meta name="generator" content="bookdown 0.18.1 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Covariance, regression, and correlation | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Covariance, regression, and correlation | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  
  
  

<meta name="author" content="Thomas Battram" />


<meta name="date" content="2020-05-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="properties-of-distributions.html"/>
<link rel="next" href="properties-of-single-loci.html"/>
<script src="libs/header-attrs-2.1.1/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="an-overview-of-quantitative-genetics.html"><a href="an-overview-of-quantitative-genetics.html"><i class="fa fa-check"></i><b>1</b> An overview of quantitative genetics</a></li>
<li class="chapter" data-level="2" data-path="properties-of-distributions.html"><a href="properties-of-distributions.html"><i class="fa fa-check"></i><b>2</b> Properties of distributions</a></li>
<li class="chapter" data-level="3" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html"><i class="fa fa-check"></i><b>3</b> Covariance, regression, and correlation</a>
<ul>
<li class="chapter" data-level="3.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#covariance"><i class="fa fa-check"></i><b>3.1</b> Covariance</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#useful-identities-for-covariance"><i class="fa fa-check"></i><b>3.1.1</b> Useful identities for covariance</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#least-squares-linear-regression"><i class="fa fa-check"></i><b>3.2</b> Least squares linear regression</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#properties-of-least-squares"><i class="fa fa-check"></i><b>3.2.1</b> Properties of least squares</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#correlation"><i class="fa fa-check"></i><b>3.3</b> Correlation</a></li>
<li class="chapter" data-level="3.4" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#differential-selection-brief-intro"><i class="fa fa-check"></i><b>3.4</b> Differential selection (brief intro)</a></li>
<li class="chapter" data-level="3.5" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#correlation-between-genotype-and-phenotype-brief-intro"><i class="fa fa-check"></i><b>3.5</b> Correlation between genotype and phenotype (brief intro)</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html"><i class="fa fa-check"></i><b>4</b> Properties of single loci</a>
<ul>
<li class="chapter" data-level="4.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#introduction"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#allele-and-genotype-frequencies"><i class="fa fa-check"></i><b>4.2</b> Allele and genotype frequencies</a></li>
<li class="chapter" data-level="4.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-transmission-of-genetic-information"><i class="fa fa-check"></i><b>4.3</b> The transmission of genetic information</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-hardy-weinberg-principle"><i class="fa fa-check"></i><b>4.3.1</b> The Hardy-Weinberg principle</a></li>
<li class="chapter" data-level="4.3.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#sex-linked-loci"><i class="fa fa-check"></i><b>4.3.2</b> Sex-linked loci</a></li>
<li class="chapter" data-level="4.3.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#polyploidy"><i class="fa fa-check"></i><b>4.3.3</b> Polyploidy</a></li>
<li class="chapter" data-level="4.3.4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#age-structure"><i class="fa fa-check"></i><b>4.3.4</b> Age structure</a></li>
<li class="chapter" data-level="4.3.5" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#testing-for-hardy-weinberg-proportions"><i class="fa fa-check"></i><b>4.3.5</b> Testing for Hardy-Weinberg proportions</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#characterising-the-influence-of-a-locus-on-the-phenotype"><i class="fa fa-check"></i><b>4.4</b> Characterising the influence of a locus on the phenotype</a></li>
<li class="chapter" data-level="4.5" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-basis-of-dominance"><i class="fa fa-check"></i><b>4.5</b> The basis of dominance</a></li>
<li class="chapter" data-level="4.6" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#fishers-decomposition-of-the-genotypic-value"><i class="fa fa-check"></i><b>4.6</b> Fisherâ€™s decomposition of the genotypic value</a></li>
<li class="chapter" data-level="4.7" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#partioning-the-genetic-variance."><i class="fa fa-check"></i><b>4.7</b> Partioning the genetic variance.</a></li>
<li class="chapter" data-level="4.8" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-effects-average-excesses-and-breeding-values"><i class="fa fa-check"></i><b>4.8</b> Additive effects, average excesses and breeding values</a></li>
<li class="chapter" data-level="4.9" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#extensions-for-multiple-alleles-and-non-random-mating"><i class="fa fa-check"></i><b>4.9</b> Extensions for multiple alleles and non random mating</a>
<ul>
<li class="chapter" data-level="4.9.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#average-excess"><i class="fa fa-check"></i><b>4.9.1</b> Average excess</a></li>
<li class="chapter" data-level="4.9.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-effects"><i class="fa fa-check"></i><b>4.9.2</b> Additive effects</a></li>
<li class="chapter" data-level="4.9.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-genetic-variance"><i class="fa fa-check"></i><b>4.9.3</b> Additive genetic variance</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html"><i class="fa fa-check"></i><b>5</b> Sources of genetic variation for multilocus traits</a>
<ul>
<li class="chapter" data-level="5.1" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#epistasis"><i class="fa fa-check"></i><b>5.1</b> Epistasis</a></li>
<li class="chapter" data-level="5.2" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#a-general-least-squares-model-for-genetic-effects"><i class="fa fa-check"></i><b>5.2</b> A general least-squares model for genetic effects</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#extension-to-haploids-and-polyploids"><i class="fa fa-check"></i><b>5.2.1</b> Extension to haploids and polyploids</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="questions.html"><a href="questions.html"><i class="fa fa-check"></i>Questions</a>
<ul>
<li class="chapter" data-level="" data-path="questions.html"><a href="questions.html#chapter-4"><i class="fa fa-check"></i>Chapter 4</a></li>
<li class="chapter" data-level="" data-path="questions.html"><a href="questions.html#chapter-5"><i class="fa fa-check"></i>Chapter 5</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="covariance-regression-and-correlation" class="section level1" number="3">
<h1><span class="header-section-number">Chapter 3</span> Covariance, regression, and correlation</h1>
<div id="covariance" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Covariance</h2>
<p>Covariance is a measure of association and the covariance between x and y would be denoted by <span class="math inline">\(\sigma(x, y)\)</span>. If x and y are independent then <span class="math inline">\(\sigma(x, y) = 0\)</span>, BUT if <span class="math inline">\(\sigma(x, y) = 0\)</span>, x and y arenâ€™t necessarily independent.</p>
<div id="useful-identities-for-covariance" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Useful identities for covariance</h3>
<p>Covariance of x with itself = variance of x:</p>
<p><span class="math display" id="eq:covxx">\[\begin{equation}
    \sigma(x, x) = \sigma^2(x)
    \tag{3.1}
\end{equation}\]</span></p>
<p>For constants (here represented by a) see <a href="covariance-regression-and-correlation.html#eq:cov-rules-contants">(3.2)</a> below
<span class="math display" id="eq:cov-rules-contants">\[\begin{align}
    \sigma(a, x) &amp;= 0 \notag \\
    \sigma(ax, y) &amp;= a\sigma(x, y) \notag \\
    \sigma^2(a, x) &amp;= a^2\sigma^2(x) \notag \\
    \sigma[(a + x), y] &amp;= \sigma(x, y) 
    \tag{3.2}
\end{align}\]</span></p>
<p>The covariance of 2 sums can be written as the sum of covariances, i.e.Â just multiply out the brackets (Iâ€™ve left this blank, do it yourself or check book):
<span class="math display">\[\sigma[(x + y),(w + z)] = ...\]</span></p>
<p>Variance of a sum is sum of variances and covariances (figure this out):
<span class="math display">\[\sigma^2(x + y) = ...\]</span></p>
</div>
</div>
<div id="least-squares-linear-regression" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Least squares linear regression</h2>
<p>Linear model:
<span class="math display">\[y = \alpha + \beta{x} + e\]</span></p>
<p>Continuing on, <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> will be the true population values and a and b will be the intercept and slope for the line of best fit derived from observed data. The derivation of <span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span> using the least-squares model can be found on pages 39-41. Buuut, who cares about that, here are the results:
<span class="math display">\[a = \bar{y} - b\bar{x}\]</span>
<span class="math display">\[b = \frac{Cov(x, y)} {Var(x)}\]</span></p>
<div id="properties-of-least-squares" class="section level3" number="3.2.1">
<h3><span class="header-section-number">3.2.1</span> Properties of least squares</h3>
<p>6 in the book, just writing down important/not obvious ones.</p>
<ul>
<li>The mean residual (<span class="math inline">\(\bar{e}\)</span>) is 0</li>
<li>Residual errors are uncorrelated with predictor variable x (see book for why)
<ul>
<li>BUT e and x may not be independent if the relationship between x and y is non-linear. If it is truly non-linear <span class="math inline">\(E(e|x) != 0\)</span></li>
</ul></li>
<li>Variance of e can vary with x, in this situation the the regression is said to display heteroscedasticity (see Figure 3.4 for great illustration)</li>
<li>The regression of y on x is different to the regression of x on y!</li>
</ul>
</div>
</div>
<div id="correlation" class="section level2" number="3.3">
<h2><span class="header-section-number">3.3</span> Correlation</h2>
<p>Correlation coefficient between x and y:
<span class="math display">\[r(x, y) = \frac{Cov(x, y)} {\sqrt{Var(x) Var(y)}}\]</span></p>
<p>The correlation coefficient is a dimensionless measure of association and it is symmetrical (i.e.Â <span class="math inline">\(r(x, y) = r(y, x)\)</span>).</p>
<p>Scaling x or y by constants does not change the correlation coefficient, but it does affect variances and covariances.</p>
<p>The correlation coefficient is a standardised regression coefficient -&gt; the regression coefficient resulting from rescaling x and y such that each has unit variance).</p>
<p><span class="math inline">\(r^2\)</span> assumes <span class="math inline">\(E(y|x)\)</span> is linear!</p>
</div>
<div id="differential-selection-brief-intro" class="section level2" number="3.4">
<h2><span class="header-section-number">3.4</span> Differential selection (brief intro)</h2>
<p>The directional selection differential, <span class="math inline">\(S\)</span>, is the difference between the mean phenotype within that generation before selection (<span class="math inline">\(\mu_s\)</span>) and the mean phenotype within that generation after (<span class="math inline">\(\mu\)</span>) selection.
<span class="math display">\[S = \mu_s - \mu\]</span></p>
<p>If all individuals have equal fertility and viability then selecting individuals wonâ€™t change anything so <span class="math inline">\(\mu_s = \mu\)</span> and <span class="math inline">\(S = 0\)</span>.</p>
<p>If <span class="math inline">\(W(z)\)</span> is the probability that individuals with phenotype <span class="math inline">\(z\)</span> survive to reproduce and <span class="math inline">\(p(z)\)</span> is the density of <span class="math inline">\(z\)</span> (pretty much means distribution) before selection, then the density after selection is:
<span class="math display">\[p_{s}(z) = \frac{W(z)p(z)} {\int W(z)p(z)dz}\]</span></p>
<p>The denominator here is the mean individual fitness (<span class="math inline">\(\bar{W}\)</span>). The relative fitness of <span class="math inline">\(z\)</span> is <span class="math inline">\(w(z) = \frac{W(z)} {\bar{W}}\)</span>.</p>
<p>After some sweet derivation (see page 46), you finish with:
<span class="math display">\[S = \sigma[z, w(z)]\]</span></p>
<p>Therefore the directional selection is equivalent to the covariance of the phenotype and the relative fitness.</p>
<p>If you regress offspring phenotype on the midparent phenotype and that relationship is linear with slope <span class="math inline">\(\beta\)</span>, a change in mean midparent phenotype induces an expected change in mean phenotype across generations equal to:
<span class="math display">\[\Delta\mu = \mu_0 - \mu = \beta(\mu_s - \mu) = \beta{S}\]</span></p>
<p>This is the breedersâ€™ equation!</p>
</div>
<div id="correlation-between-genotype-and-phenotype-brief-intro" class="section level2" number="3.5">
<h2><span class="header-section-number">3.5</span> Correlation between genotype and phenotype (brief intro)</h2>
<p>Only when there is no gene-environment interaction is the variance explained by genetics (broad-sense heritability) the equation below:
<span class="math display">\[H^2 = \frac{\sigma^2_G} {\sigma^2_z}\]</span>,</p>
<p>where <span class="math inline">\(z\)</span> is the phenotype and G is the sum of the total effects (not just additive) at all loci on the trait.</p>
<p>The slope of a midparent-offspring regression provides an estimate of the proportion of the phenotypic variance that is attributable to additive genetic factors (the narrow-sense heritability).
<span class="math display">\[h^2 = \frac{\sigma^2_A} {\sigma^2_z}\]</span></p>
<p>So as <span class="math inline">\(h^2\)</span> is just the regression of offspring phenotype on midparent phenotype it can actually be used in the breedersâ€™ equation!
<span class="math display">\[\Delta\mu = h^2S\]</span></p>
<p>So the narrow-sense heritability can be thought of as the efficiency of the response to selection. If <span class="math inline">\(h^2 = 0\)</span> there can be no evolutionary change regardless of strength of selection. Although this should be obvious because if <span class="math inline">\(h^2\)</span> is 0 then there is clearly no passing of genetic material onto the next generation that is influencing that trait.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="properties-of-distributions.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="properties-of-single-loci.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["quant_genetics_book_notes.pdf", "quant_genetics_book_notes.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
