<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Resemblance between relatives | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits</title>
  <meta name="description" content="Chapter 7 Resemblance between relatives | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  <meta name="generator" content="bookdown 0.18.1 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Resemblance between relatives | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Resemblance between relatives | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  
  
  

<meta name="author" content="Thomas Battram" />


<meta name="date" content="2020-06-07" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="sources-of-environmental-variation.html"/>
<link rel="next" href="questions.html"/>
<script src="libs/header-attrs-2.1.1/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="an-overview-of-quantitative-genetics.html"><a href="an-overview-of-quantitative-genetics.html"><i class="fa fa-check"></i><b>1</b> An overview of quantitative genetics</a></li>
<li class="chapter" data-level="2" data-path="properties-of-distributions.html"><a href="properties-of-distributions.html"><i class="fa fa-check"></i><b>2</b> Properties of distributions</a></li>
<li class="chapter" data-level="3" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html"><i class="fa fa-check"></i><b>3</b> Covariance, regression, and correlation</a>
<ul>
<li class="chapter" data-level="3.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#covariance"><i class="fa fa-check"></i><b>3.1</b> Covariance</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#useful-identities-for-covariance"><i class="fa fa-check"></i><b>3.1.1</b> Useful identities for covariance</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#least-squares-linear-regression"><i class="fa fa-check"></i><b>3.2</b> Least squares linear regression</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#properties-of-least-squares"><i class="fa fa-check"></i><b>3.2.1</b> Properties of least squares</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#correlation"><i class="fa fa-check"></i><b>3.3</b> Correlation</a></li>
<li class="chapter" data-level="3.4" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#differential-selection-brief-intro"><i class="fa fa-check"></i><b>3.4</b> Differential selection (brief intro)</a></li>
<li class="chapter" data-level="3.5" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#correlation-between-genotype-and-phenotype-brief-intro"><i class="fa fa-check"></i><b>3.5</b> Correlation between genotype and phenotype (brief intro)</a></li>
<li class="chapter" data-level="3.6" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#end-of-chapter-questions"><i class="fa fa-check"></i><b>3.6</b> End of chapter questions</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html"><i class="fa fa-check"></i><b>4</b> Properties of single loci</a>
<ul>
<li class="chapter" data-level="4.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#introduction"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#allele-and-genotype-frequencies"><i class="fa fa-check"></i><b>4.2</b> Allele and genotype frequencies</a></li>
<li class="chapter" data-level="4.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-transmission-of-genetic-information"><i class="fa fa-check"></i><b>4.3</b> The transmission of genetic information</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-hardy-weinberg-principle"><i class="fa fa-check"></i><b>4.3.1</b> The Hardy-Weinberg principle</a></li>
<li class="chapter" data-level="4.3.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#sex-linked-loci"><i class="fa fa-check"></i><b>4.3.2</b> Sex-linked loci</a></li>
<li class="chapter" data-level="4.3.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#polyploidy"><i class="fa fa-check"></i><b>4.3.3</b> Polyploidy</a></li>
<li class="chapter" data-level="4.3.4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#age-structure"><i class="fa fa-check"></i><b>4.3.4</b> Age structure</a></li>
<li class="chapter" data-level="4.3.5" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#testing-for-hardy-weinberg-proportions"><i class="fa fa-check"></i><b>4.3.5</b> Testing for Hardy-Weinberg proportions</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#characterising-the-influence-of-a-locus-on-the-phenotype"><i class="fa fa-check"></i><b>4.4</b> Characterising the influence of a locus on the phenotype</a></li>
<li class="chapter" data-level="4.5" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-basis-of-dominance"><i class="fa fa-check"></i><b>4.5</b> The basis of dominance</a></li>
<li class="chapter" data-level="4.6" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#fishers-decomposition-of-the-genotypic-value"><i class="fa fa-check"></i><b>4.6</b> Fisher’s decomposition of the genotypic value</a></li>
<li class="chapter" data-level="4.7" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#partioning-the-genetic-variance."><i class="fa fa-check"></i><b>4.7</b> Partioning the genetic variance.</a></li>
<li class="chapter" data-level="4.8" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-effects-average-excesses-and-breeding-values"><i class="fa fa-check"></i><b>4.8</b> Additive effects, average excesses and breeding values</a></li>
<li class="chapter" data-level="4.9" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#extensions-for-multiple-alleles-and-non-random-mating"><i class="fa fa-check"></i><b>4.9</b> Extensions for multiple alleles and non random mating</a>
<ul>
<li class="chapter" data-level="4.9.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#average-excess"><i class="fa fa-check"></i><b>4.9.1</b> Average excess</a></li>
<li class="chapter" data-level="4.9.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-effects"><i class="fa fa-check"></i><b>4.9.2</b> Additive effects</a></li>
<li class="chapter" data-level="4.9.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-genetic-variance"><i class="fa fa-check"></i><b>4.9.3</b> Additive genetic variance</a></li>
</ul></li>
<li class="chapter" data-level="4.10" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#end-of-chapter-questions-1"><i class="fa fa-check"></i><b>4.10</b> End of chapter questions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html"><i class="fa fa-check"></i><b>5</b> Sources of genetic variation for multilocus traits</a>
<ul>
<li class="chapter" data-level="5.1" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#epistasis"><i class="fa fa-check"></i><b>5.1</b> Epistasis</a></li>
<li class="chapter" data-level="5.2" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#a-general-least-squares-model-for-genetic-effects"><i class="fa fa-check"></i><b>5.2</b> A general least-squares model for genetic effects</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#extension-to-haploids-and-polyploids"><i class="fa fa-check"></i><b>5.2.1</b> Extension to haploids and polyploids</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#linkage"><i class="fa fa-check"></i><b>5.3</b> Linkage</a></li>
<li class="chapter" data-level="5.4" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#effect-of-disequilibrium-of-the-genetic-variance"><i class="fa fa-check"></i><b>5.4</b> Effect of disequilibrium of the genetic variance</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#the-evidence"><i class="fa fa-check"></i><b>5.4.1</b> The evidence</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#end-of-chapter-questions-2"><i class="fa fa-check"></i><b>5.5</b> End of chapter questions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html"><i class="fa fa-check"></i><b>6</b> Sources of Environmental Variation</a>
<ul>
<li class="chapter" data-level="6.1" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html#extension-of-the-linear-model-to-phenotypes"><i class="fa fa-check"></i><b>6.1</b> Extension of the linear model to phenotypes</a></li>
<li class="chapter" data-level="6.2" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html#special-environmental-effects"><i class="fa fa-check"></i><b>6.2</b> Special environmental effects</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html#within-individual-variation"><i class="fa fa-check"></i><b>6.2.1</b> Within-individual variation</a></li>
<li class="chapter" data-level="6.2.2" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html#developmental-homeostasis-and-homozygosity"><i class="fa fa-check"></i><b>6.2.2</b> Developmental homeostasis and homozygosity</a></li>
<li class="chapter" data-level="6.2.3" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html#repeatability"><i class="fa fa-check"></i><b>6.2.3</b> Repeatability</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html#general-environmental-effects-of-maternal-origin"><i class="fa fa-check"></i><b>6.3</b> General environmental effects of maternal origin</a></li>
<li class="chapter" data-level="6.4" data-path="sources-of-environmental-variation.html"><a href="sources-of-environmental-variation.html#genotype-x-environment-interaction"><i class="fa fa-check"></i><b>6.4</b> Genotype x environment interaction</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html"><i class="fa fa-check"></i><b>7</b> Resemblance between relatives</a>
<ul>
<li class="chapter" data-level="7.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#measures-of-relatedness"><i class="fa fa-check"></i><b>7.1</b> Measures of relatedness</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#coefficients-of-identity"><i class="fa fa-check"></i><b>7.1.1</b> Coefficients of identity</a></li>
<li class="chapter" data-level="7.1.2" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#coefficients-of-coancestry-and-inbreeding"><i class="fa fa-check"></i><b>7.1.2</b> Coefficients of coancestry and inbreeding</a></li>
<li class="chapter" data-level="7.1.3" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#the-coefficient-of-fraternity"><i class="fa fa-check"></i><b>7.1.3</b> The coefficient of fraternity</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#the-genetic-covariance-between-relatives"><i class="fa fa-check"></i><b>7.2</b> The genetic covariance between relatives</a></li>
<li class="chapter" data-level="7.3" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#the-effect-of-linkage-and-gametic-phase-disequilibrium"><i class="fa fa-check"></i><b>7.3</b> The effect of linkage and gametic phase disequilibrium</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#gametic-phase-disequilibrium"><i class="fa fa-check"></i><b>7.3.1</b> Gametic phase disequilibrium</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#assortative-mating"><i class="fa fa-check"></i><b>7.4</b> Assortative mating</a></li>
<li class="chapter" data-level="7.5" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#polyploidy-1"><i class="fa fa-check"></i><b>7.5</b> Polyploidy</a></li>
<li class="chapter" data-level="7.6" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#environmental-sources-of-covariance-between-relatives"><i class="fa fa-check"></i><b>7.6</b> Environmental sources of covariance between relatives</a></li>
<li class="chapter" data-level="7.7" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#the-heritability-concept"><i class="fa fa-check"></i><b>7.7</b> The heritability concept</a>
<ul>
<li class="chapter" data-level="7.7.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#evolvability"><i class="fa fa-check"></i><b>7.7.1</b> Evolvability</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="questions.html"><a href="questions.html"><i class="fa fa-check"></i>Questions</a>
<ul>
<li class="chapter" data-level="" data-path="questions.html"><a href="questions.html#chapter-4"><i class="fa fa-check"></i>Chapter 4</a></li>
<li class="chapter" data-level="" data-path="questions.html"><a href="questions.html#chapter-5"><i class="fa fa-check"></i>Chapter 5</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="resemblance-between-relatives" class="section level1" number="7">
<h1><span class="header-section-number">Chapter 7</span> Resemblance between relatives</h1>
<p>If you ignore GxE you can express the phenotypic values of individuals <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span> (recall equation <a href="sources-of-environmental-variation.html#eq:phenotype-linear-model">(6.1)</a>) simply as <span class="math inline">\(Z_x = G_x + E_x + e_x\)</span> and <span class="math inline">\(Z_y = G_y + E_y + e_y\)</span>. This chapter is interested in the resemblence between relatives, so using these equations we can specify what the covariance between phenotypic values will be:</p>
<p><span class="math display" id="eq:phenotypic-covariance-no-gxe">\[\begin{equation}
\begin{split}
  \sigma_z(x, y) &amp;= \sigma[(G_x + E_x + e_x), (G_y + E_y + e_y)] \\
  &amp;= \sigma_G(x, y) + \sigma_{G.E}(x, y) + \sigma_{G.E}(y, x) + \sigma_{E}(x, y)
\end{split}
\tag{7.1}
\end{equation}\]</span></p>
<p>Remember, <span class="math inline">\(e\)</span> (special environmental effects) are derived from random residual deviations so are uncorrelated between individuals (think within-individual variation). You can design experiments so all terms with <span class="math inline">\(E\)</span> in them have expected values of 0 and here we’re going to assume that one individual’s genotypic effects are not covarying with the others general environmental effects, i.e. <span class="math inline">\(\sigma_{G.E}(x, y) = \sigma_{G.E}(y, x) = 0\)</span>. This boils everything nicely down to this simple equation:</p>
<p><span class="math display" id="eq:phenotypic-covariance-no-gxe-simple">\[\begin{equation}
  \sigma_z(x, y) = \sigma_G(x, y) + \sigma_{E}(x, y)
  \tag{7.2}
\end{equation}\]</span></p>
<p><span class="math inline">\(\sigma_G(x, y)\)</span> will be the focus of things to come! Like genetic variance, the covariance can be split into components attributable to additive, dominance, and epistatic effects. Each term is simply one of the terms used to describe genetic variance (e.g. equation <a href="sources-of-genetic-variation-for-multilocus-traits.html#eq:multilocus-genetic-variance">(5.6)</a>), weighted by a coefficient that describes the joint distribution of effects in pairs of relatives.</p>
<p>Complications of estimating these coefficients include, non-random mating, LD, assortative mating, sex-linkage, maternal genetic effects and inbreeding.</p>
<div id="measures-of-relatedness" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Measures of relatedness</h2>
<p>Relatedness can only be defined with respect to a specified frame of reference as all individuals are related (DUH). From here on the reference population is the base of the observed pedigree. So if the observed data is just trios, then the base population is the parents in those trios. If, data on grandparents is observed then they’re the base population and so-on. Members of the base population are assumed to be unrelated. Also when discussing relatedness we refer to identity by descent (IBD), not identity by state (IBS).</p>
<hr />
<p><strong>Identity by descent and identity by state</strong></p>
<p>Genes that are identical by descent are those that have been passed down by a common ancestor. The same gene from two individuals may share the same genetic sequence, making them identical by state, but if they do not derive from the same common ancestor they are not identical by descent.</p>
<p>So, genes that are identical by descent must, except for mutations, be identical by state, BUT genes that are identical by state might not be identical by descent.</p>
<hr />
<div id="coefficients-of-identity" class="section level3" number="7.1.1">
<h3><span class="header-section-number">7.1.1</span> Coefficients of identity</h3>
<p>At a single locus in a diploid individual there are two alleles so with two individuals you have four alleles. Each allele is inherited singularly (a gamete only passes on one copy), so has it’s own identity with each of the other three alleles. This means identity within individuals and between individuals can exist. This scenario gives rise to 15 different configurations of identity by descent. Individuals that contain pairs of alleles that are identical by descent are said to be inbred. Ignoring difference between maternally and paternally derived alleles, the number of IBD configurations reduces to nine. These range from a state where all four alleles are identical by descent (two inbred individuals that share a common ancestor) to a state where none of the alleles are identical by descent. In a large population with randomly mating individuals most states don’t exist. The probabilities associated with each of the nine states are called the condensed coefficients of identity. Consider the case of a single gene for two non-inbred full sibs. There is a probability of 0.5 that both sibs inherit the same allele from their father and, independently, the probability they inherit the same allele from their mother is 0.5. So there is a probability of 0.25 that both pairs of alleles are identical by descent (i.e. the alleles inherited from the mother were the same and the alleles inherited from the father were the same.), there is a probability of 0.5 that just one pair is identical by descent and a probability of 0.25 that neither pair are identical by descent. All other states have a probability of 0.</p>
</div>
<div id="coefficients-of-coancestry-and-inbreeding" class="section level3" number="7.1.2">
<h3><span class="header-section-number">7.1.2</span> Coefficients of coancestry and inbreeding</h3>
<p>Suppose single genes (or alleles – side note alleles and genes will probably be used interchangably in this chapter) are drawn randomly from individuals <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>. The probability that these two genes are identical by descent, <span class="math inline">\(\Theta_{xy}\)</span>, is the coefficient of coancestry (can be called coefficient of consanguinity or coefficient of kinship). See figure 7.2 in the book for a graphical depiction of the nine IBD state classes and equation 7.2 in the book relates these states to <span class="math inline">\(\Theta_{xy}\)</span> –&gt; each state is weighted by the conditional probability that a randomly drawn gene from <span class="math inline">\(x\)</span> is identical by descent with a randomly drawn gene from <span class="math inline">\(y\)</span>.</p>
<p>For an individual, <span class="math inline">\(z\)</span>, their inbreeding coefficient (<span class="math inline">\(f_z\)</span>) is equal to their parents coefficient of coancestry (<span class="math inline">\(f_z = \Theta_{xy}\)</span>).</p>
<p>To derive <span class="math inline">\(\Theta_{xy}\)</span>, we first need to derive <span class="math inline">\(\Theta_{xx}\)</span> (may seem weird, but it ain’t). If you took a gene with two alleles, <span class="math inline">\(A_1\)</span> and <span class="math inline">\(A_2\)</span> and you could know which parent each came from to distinguish them, if you drew one allele at random then replaced it and drew another you could draw <span class="math inline">\(A_1\)</span> twice, <span class="math inline">\(A_1\)</span> then <span class="math inline">\(A_2\)</span>, <span class="math inline">\(A_2\)</span> then <span class="math inline">\(A_1\)</span> or <span class="math inline">\(A_2\)</span> twice. If they’re not copies of the same allele (i.e. <span class="math inline">\(A_1\)</span> doesn’t equal <span class="math inline">\(A_2\)</span>) then if <span class="math inline">\(A_1\)</span> is drawn twice it must be identical by descent and the same goes for <span class="math inline">\(A_2\)</span>. In this scenario, <span class="math inline">\(\Theta_{xx} = (1/4)(1) + (1/4)(1)\)</span>. Of course, the individual could be inbred and so the probability that <span class="math inline">\(A_1\)</span> and <span class="math inline">\(A_2\)</span> are identical by descent is <span class="math inline">\(f_x\)</span>. A general expression for the coefficient of coancestry of an individual with itself is given below in equation <a href="resemblance-between-relatives.html#eq:coef-coan-itself">(7.3)</a></p>
<p><span class="math display" id="eq:coef-coan-itself">\[\begin{equation}
  \Theta_{xx} = \frac{1}{4}(1 + f_x + f_x + 1) = \frac{1}{2}(1 + f_x)
  \tag{7.3}
\end{equation}\]</span></p>
<p>Something to note, the fact this coefficient increases with inbreeding is used (kind of) in current SNP-heritability estimations. After creating a kinship matrix, the diagonal of the matrix represents <span class="math inline">\(2\Theta_{xx}\)</span>. Due to the software, it will always be centred on one (mean of one) and without inbreeding expect a normal distribution with small variation around the mean, and with inbreeding this distribution would be right-skewed and you can identify inbred individuals by assessing which individuals diagonal elements are significantly larger than one.</p>
<p>Parent (<span class="math inline">\(p\)</span>) and offspring (<span class="math inline">\(o\)</span>) scenario now!! If neither are inbred (<span class="math inline">\(p\)</span>’s parents unrelated and they are unrelated to their mate), then that makes things simple. When drawing one of the two alleles from the mother and then one of the two alleles from the offspring, there is only one scenario in which they are the same. As each scenario has an equal probability of occurring, <span class="math inline">\(\Theta_{po} = \frac{1}{4}\)</span>. If <span class="math inline">\(p\)</span> is inbred, probability of their alleles being identical by descent is <span class="math inline">\(f_p\)</span>. This is the same as the probability of the offspring allele being identical by descent to the maternal allele the offspring did not inherit. Probability of drawing inherited allele from offspring and allele not passed on from parent is (like the others) 1/4. Therfore, inbreeding inflates the coefficient of coancestry to <span class="math inline">\(\Theta_{po} = \frac{f_p}{4} + \frac{1}{4} = \frac{1 + f_p}{4}\)</span>. Complete inbreeding means <span class="math inline">\(f_p = 1\)</span> so <span class="math inline">\(\Theta_{po} = \frac{1}{2}\)</span>. By thinking about probability of picking paternally derived allele and stuff you can add in <span class="math inline">\(f_o\)</span> (see book page 136) and the general expression is given in equation <a href="resemblance-between-relatives.html#eq:coef-coan-parent-offspring">(7.4)</a> below.</p>
<p><span class="math display" id="eq:coef-coan-parent-offspring">\[\begin{equation}
  \Theta_{po} = \frac{1}{4}(1 + f_p + 2f_o)
  \tag{7.4}
\end{equation}\]</span></p>
<p>Often in the literature <span class="math inline">\(\Theta_{po}\)</span> is considered to simply be 1/4. So no inbreeding is assumed.</p>
<p>Full sibs time! <span class="math inline">\(m\)</span> = mother, <span class="math inline">\(f\)</span> = father, <span class="math inline">\(x\)</span> = kid1 and <span class="math inline">\(y\)</span> = kid2. So if <span class="math inline">\(m\)</span> and <span class="math inline">\(f\)</span> are not inbred or related themselves then there are two situations from which a child could inherit the same allele by descent. Either that allele has come from <span class="math inline">\(m\)</span> or that allele has come from <span class="math inline">\(f\)</span>. Both have the same probability so let’s just look at <span class="math inline">\(m\)</span> -&gt; <span class="math inline">\(x\)</span> and <span class="math inline">\(m\)</span> -&gt; <span class="math inline">\(y\)</span>. The probability <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span> receive the same maternal allele is 1/2 (i.e. the coefficient of coancestry of the mum with herself, <span class="math inline">\(\Theta_{mm}\)</span>). The probability of randomly drawing the maternally inherited allele from <span class="math inline">\(x\)</span> is 1/2 and the same is true for <span class="math inline">\(y\)</span>. Therefore the probability of drawing one allele from <span class="math inline">\(x\)</span> and one from <span class="math inline">\(y\)</span>, that are identical by descent, passed down from <span class="math inline">\(m\)</span> is <span class="math inline">\(\Theta_{mm}/4 = 1/8\)</span>. Adding contribution from el dado, we get <span class="math inline">\(\Theta_{xy} = 1/4\)</span> EASY! Appendix 2 contains path analysis, developed by Sewall Wright, that can derive these results. Now we allow for inbreeding of parents so introduce <span class="math inline">\(f_m\)</span> and <span class="math inline">\(f_f\)</span>. There are still only two paths that lead to alleles identical by descent in <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span> (they’re just more likely with inbreeding because alleles within the father and within the mother are more likely to be identical by descent, i.e. <span class="math inline">\(\Theta_{mm}\)</span> &gt; 1/2 and so is <span class="math inline">\(\Theta_{ff}\)</span> if there is inbreeding). Including these terms gives:</p>
<p><span class="math display">\[\begin{equation}
  \Theta_{xy} = \frac{1}{4}(\Theta_{mm} + \Theta_{ff}) = \frac{1}{4}(\frac{1 + f_m}{2} + \frac{1 + f_f}{2}) = \frac{1}{8}(2 + f_m + f_f) \notag
\end{equation}\]</span></p>
<p>By taking into account inbreeding coefficients of kids, we end up with:</p>
<p><span class="math display" id="eq:coef-coan-full-sibs">\[\begin{equation}
  \Theta_{xy} = \frac{1}{8}(2 + f_m + f_f + 4\Theta_{mf}) \notag
  \tag{7.5}
\end{equation}\]</span></p>
<p>Under random mating <span class="math inline">\(\Theta_{xy} = 1/4\)</span>.</p>
<p>These techniques can be extended to more distant relatives and more complicated schemes of relatedness. The coefficient of coancestry is always the sum of a series of two types of paths between <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>. The first type of path leads from a single common ancestor to the two individuals of interest, while the second type passes through two remote ancestors that are related to each other. Neither type of path is allowed to pass through the same ancestor more than once. This procedure is summarised by equation <a href="resemblance-between-relatives.html#eq:coef-coan-xy-autosomes">(7.6)</a> below.</p>
<p><span class="math display" id="eq:coef-coan-xy-autosomes">\[\begin{equation}
  \Theta_{xy} = \sum_{i}\Theta_{ii}(\frac{1}{2})^{n_i - 1} + \sum_{j}\sum_{j \ne k}\Theta_{jk}(\frac{1}{2})^{n_{jk} - 2}
  \tag{7.6}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(n_i\)</span> is the number of individuals (including <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>) in the path leading from common ancestor <span class="math inline">\(i\)</span>, and <span class="math inline">\(n_{jk}\)</span> is the number of individuals (including <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>) on the path leading from two different but related ancestors, <span class="math inline">\(j\)</span> and <span class="math inline">\(k\)</span>.</p>
<p>Been assuming autosomal genes until this point! Sex-linked genes means we have to change things a bit. See book for deets (only 1 paragraph).</p>
</div>
<div id="the-coefficient-of-fraternity" class="section level3" number="7.1.3">
<h3><span class="header-section-number">7.1.3</span> The coefficient of fraternity</h3>
<p>Before we were focusing on single alleles, but it is also of importance to consider the probability that two individuals (again <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>) contain genotypes that are identical by descent, this is called the coefficient of fraternity, <span class="math inline">\(\Delta_{xy}\)</span>. Mother of <span class="math inline">\(x\)</span> = <span class="math inline">\(m_x\)</span> and father of <span class="math inline">\(y\)</span> = <span class="math inline">\(f_y\)</span> etc. There are four combinations of parents (excluding the current combination). There are two ways in which the genotype of <span class="math inline">\(x\)</span> can match the genotype of <span class="math inline">\(y\)</span> (1) If allele from <span class="math inline">\(m_x\)</span> and <span class="math inline">\(m_y\)</span> are identical by descent and the allele from <span class="math inline">\(f_x\)</span> and <span class="math inline">\(f_y\)</span> are identical by descent (2) allele from <span class="math inline">\(m_x\)</span> is identical by descent with allele from <span class="math inline">\(f_y\)</span> and allele from <span class="math inline">\(f_x\)</span> is identical by descent with that from <span class="math inline">\(m_y\)</span>. So the coefficient of fraternity can be defined as</p>
<p><span class="math display" id="eq:coef-frat">\[\begin{equation}
  \Delta_{xy} = \Theta_{m_{x}m_{y}}\Theta_{f_{x}f_{y}} + \Theta_{m_{x}f_{y}}\Theta_{f_{x}m_{y}}
  \tag{7.7}
\end{equation}\]</span></p>
<p>For full sibs (<span class="math inline">\(m_x\)</span> = <span class="math inline">\(m_y\)</span> = <span class="math inline">\(m\)</span> and <span class="math inline">\(f_x\)</span> = <span class="math inline">\(f_y\)</span> = <span class="math inline">\(f\)</span>) equation <a href="resemblance-between-relatives.html#eq:coef-frat">(7.7)</a> reduces to:</p>
<p><span class="math display" id="eq:coef-frat-full-sibs">\[\begin{equation}
  \Delta_{xy} = \Theta_{mm}\Theta_{ff} + \Theta^2_{mf}
  \tag{7.8}
\end{equation}\]</span></p>
<p>If the parents are unrelated <span class="math inline">\(\Theta_{mf} = 0\)</span>; and if the parents are not inbred <span class="math inline">\(\Theta_{mm} = \Theta_{ff} = 1/2\)</span> so <span class="math inline">\(\Delta_{xy} = 1/4\)</span>. Now for paternal half-sibs, where fathers are the same (i.e. <span class="math inline">\(f_x\)</span> = <span class="math inline">\(f_y\)</span> = <span class="math inline">\(f\)</span>).</p>
<p><span class="math display" id="eq:coef-frat-half-sibs">\[\begin{equation}
  \Delta_{xy} = \Theta_{m_{x}m_{y}}\Theta_{ff} + \Theta_{m_{x}f}\Theta_{fm_{y}}
  \tag{7.9}
\end{equation}\]</span></p>
<p>If parents are unrelated then <span class="math inline">\(\Theta_{ff} = 1/2\)</span> and <span class="math inline">\(\Theta_{m_{x}m_{y}} = \Theta_{m_{x}f} = \Theta_{fm_{y}} = 0\)</span> so <span class="math inline">\(\Delta_{xy} = 0\)</span>. So the genotypes of two individuals can’t be identical by descent if their maternally (or paternally) derived alleles come from unrelated individuals.</p>
</div>
</div>
<div id="the-genetic-covariance-between-relatives" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> The genetic covariance between relatives</h2>
<p>To assess genetic covariance between individuals we must use what we learnt from partitioning genetic variance in Chapter 5. To start with we’ll make a lot of assumptions:</p>
<ol style="list-style-type: decimal">
<li>diploid autosomal loci only</li>
<li>random mating</li>
<li>loci are unlinked and are not in LD</li>
<li>no genetic variation for maternal effects</li>
<li>no GxE or genotype-environment covariance</li>
<li>no sexual dimorphism</li>
<li>no selection</li>
</ol>
<p>Recalling equation <a href="sources-of-genetic-variation-for-multilocus-traits.html#eq:complete-genotypic-value">(5.5)</a>, we can see the complete genotypic value for two loci, which will be the same for individuals <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span>. The multilocus genetic variance for each individual is then given in equation <a href="sources-of-genetic-variation-for-multilocus-traits.html#eq:multilocus-genetic-variance">(5.6)</a>. As different types of effects (e.g. additive and dominance) are uncorrelated within individuals, they’re also uncorrelated between individuals. So the genetic covariance is simply:</p>
<p><span class="math display" id="eq:multilocus-genetic-covariance">\[\begin{equation}
  \sigma_G(x, y) = \sigma_A(x, y) + \sigma_D(x, y) + \sigma_{AA}(x, y) + \sigma_{AD}(x, y) + \sigma_{DD}(x, y) + ...
  \tag{7.10}
\end{equation}\]</span></p>
<p>Remember <span class="math inline">\(\sigma^2(x) == \sigma(x,x)\)</span></p>
<p>Then the book expresses terms in equation <a href="resemblance-between-relatives.html#eq:multilocus-genetic-covariance">(7.10)</a> in terms of variance components and coefficients of relationships. Confusing, come back to this!</p>
<p>The genetic covariance between relatives becomes (equation 7.12 in book):</p>
<p><span class="math display" id="eq:multilocus-genetic-covariance-relatives">\[\begin{equation}
\begin{split}
  \sigma_{G}(x, y) &amp;= \sum(2\Theta_{xy})^n \Delta^{m}_{xy}\sigma^2_{A^{n}D^{m}} \\
  &amp; = 2\Theta_{xy}\sigma^2_A + \Delta_{xy}\sigma^2_{D} + (2\Theta_{xy})^2\sigma^2_{AA} + 2\Theta_{xy}\Delta_{xy}\sigma^2_{AD} + \Delta^2_{xy}\sigma^2_{DD} + (2\Theta_{xy})^3\sigma^2_{AAA} + ...
\end{split}
\tag{7.11}
\end{equation}\]</span></p>
<p>Using values from Table 7.1 in the book you can then derive explicit expressions for the genetic covariances among common types of relative - these are in Table 7.2!</p>
<p>These values show some interesting things:</p>
<ul>
<li>Gene action involving dominance only rarely contributes to the covariance between relatives. It requires that each parent of <span class="math inline">\(x\)</span> be related to a different parent of <span class="math inline">\(y\)</span>. Such relationships (full sibs, double first cousins, and monozygotic twins) are said to be collateral.</li>
<li>The coefficient for ^2_{AA} declines more rapidly with the distance of the relationship than does that for ^2_{A}.</li>
</ul>
<p>In the same way that the lack of variance that dominance and epistatic effects contribute to total genetic variance, doesn’t tell us anything about how the genes being measured function, the lack of variance dominance and epistatic effects contribute to the genetic covariance between relatives tells us nothing about how the function of the measured genes contributes to resemblance between relatives.</p>
<p>WRITE THESE DOWN!</p>
</div>
<div id="the-effect-of-linkage-and-gametic-phase-disequilibrium" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> The effect of linkage and gametic phase disequilibrium</h2>
<p>Under the assumption of linkage equilibrium, linkage influences on the epistatic components of genetic covariance.</p>
<p>COME BACK TO THIS SECTION… if only epistatic components relavent, then I don’t really care tbh</p>
<div id="gametic-phase-disequilibrium" class="section level3" number="7.3.1">
<h3><span class="header-section-number">7.3.1</span> Gametic phase disequilibrium</h3>
<p>Somehow (work this out later), the book extrapolates from equations <a href="sources-of-genetic-variation-for-multilocus-traits.html#eq:additve-var-LD">(5.11)</a> and <a href="sources-of-genetic-variation-for-multilocus-traits.html#eq:dominance-var-LD">(5.12)</a> to give the total genetic variance in the presence of ld:</p>
<p><span class="math display" id="eq:total-genetic-variance-ld">\[\begin{equation}
    \sigma^2_{G} = \sigma^2_{A} + \sigma_{A,A} + \sigma^2_{D} + \sigma_{D,D}
    \tag{7.12}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(\sigma_{A,A}\)</span> is the contribution due to covariance of additive effects of nonalleles within gametes (the additive disequilibrium covariance), and <span class="math inline">\(\sigma_{D,D}\)</span> is the contribution of covariance due to dominance effects of different loci within individuals (the dominance disequilibrium covariance).</p>
<p>For situations in which the study population is being maintained in a steady state of disequilibrium from generation to generation (by processes such as natural selection, migration, and/or non-random mating), the modifications needed to estimate genetic covariance between relatives is simple - just reokace <span class="math inline">\(\sigma^2_{A}\)</span> with <span class="math inline">\(\sigma^2_{A} + \sigma_{A,A}\)</span> and <span class="math inline">\(\sigma^2_{D}\)</span> with <span class="math inline">\(\sigma^2_{D} + \sigma_{D,D}\)</span>. However, when mating is random and forces acting on the maintenance of disequilibria are dampened, then covariance between relatives changes with time as the recombination frequency (REMEMBER THAT BAD BOY FROM CHAPTER 5) causes a gradual decay of disequilibrium.</p>
<p>Now we consider two loci, remember the pair of alleles (here <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> or <span class="math inline">\(k\)</span> and <span class="math inline">\(l\)</span>) are inherited from different gametes. The additive genetic variance is simply:</p>
<p><span class="math display" id="eq:additive-genetic-variance-as-expectations">\[\begin{equation}
    \sigma^2_{A} = E(\alpha^2_i) + E(\alpha^2_j) + E(\alpha^2_k) + E(\alpha^2_l)
    \tag{7.13}
\end{equation}\]</span></p>
<p>additive disequilibrium covariance in the base population is:</p>
<p><span class="math display" id="eq:additive-disequilibrium-covariance-as-expectations">\[\begin{equation}
    \sigma_{A,A}(0) = 2E(\alpha_i\alpha_k) + 2E(\alpha_j\alpha_l)
    \tag{7.14}
\end{equation}\]</span></p>
<p>Under the assumption of random mating <span class="math inline">\(E(\alpha_i\alpha_j)\)</span> and <span class="math inline">\(E(\alpha_k\alpha_l)\)</span> would be zero because there should be no correlation between the gametes of random, unrelated mates. As we’re assuming no forces are acting to maintain ld between alleles, we can assume it decays as given by <a href="sources-of-genetic-variation-for-multilocus-traits.html#eq:LD-decay">(5.9)</a>. Thus, the decay of the additive disequilibrium covariance can be given by:</p>
<p><span class="math display" id="eq:additive-disequilibrium-covariance-decay">\[\begin{equation}
  \sigma_{A,A}(t) = (1 - c)^{t}\sigma_{A,A}(0)
  \tag{7.15}
\end{equation}\]</span></p>
<p>Unclear how the book then gets equation 7.15D, but then goes on to say that the additive disequilibrium covariance between relatives is given by:</p>
<p><span class="math display" id="eq:additive-disequilibrium-covariance-between-relatives">\[\begin{equation}
  \sigma_{A,A}(x, y, t) = 2\Theta_{xy}(1 - c)^{t}\sigma_{A,A}(0)
  \tag{7.16}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(t\)</span> denotes the number of generations that the common ancestors are removed from the base population. Thus, for example, the parent-offspring covariance (_{xy} = 1/4) resulting from additive disequilibrium covariance is <span class="math inline">\(\sigma_{A,A}(0)/2\)</span> if the parents are from the base population. Then for each generation down the parents are thne multiply that by <span class="math inline">\((1 - c)\)</span>, so if they were fourth generation it’d be <span class="math inline">\((1-c)^3\sigma_{A,A}(0)/2\)</span>. Derivation of the covariance between relatives resulting from dominance disequilibrium covariance follows the same logic just presented. However, in this case, the dominance disequilibrium covariance declines each generation to <span class="math inline">\((1 - c)^2\)</span> it’s previous value. See book for why COME BACK TO THIS!</p>
<p>See Table 7.3 in the book for disequilibrium covariances of the base-population among relatives. COME BACK TO FINAL PAGE OF THIS SECTION!</p>
</div>
</div>
<div id="assortative-mating" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> Assortative mating</h2>
<p>Assortative mating isn’t uncommon, especially positive assortative mating, whereby individuals choose mates that have phenotypes that resemble their own. Lots of examples of this in human populations.</p>
<p>Many assortative mating systems are selective, such that some phenotypes of one or both sexes have a greater ability to attract mates than others, but this is beyond the scope of this book. Here we assume all individuals have an equal opportunity to reproduce, the only limitation being the phenotypic distribution of available mates (i.e. nonselective assortative mating). Difference between inbreeding and assortative mating: inbreeding is choice of mates by similar genotypes and assortative mating is choice of mate by similar phenotypes. Of course, if the trait being assorted on has some genetic component (i.e. every fucking trait), then assortative mating will increase the chances that mates share identical alleles, BUT in all practical scenarios (especially for humans), it won’t cause extreme genetic structures like a completely homozygous population, like inbreeding can.</p>
<p>Inbreeding can inflate the genetic variance of a population up to twofold in the abscence of epistasis (<a href="properties-of-single-loci.html#properties-of-single-loci">Properties of single loci</a>), but strong assortative mating can cause an even larger increase (of course disassortative mating induces a reduction of the genetic variance). Positive assortative mating increases the coupling of genes with similar effects.</p>
<p><strong>How assortative mating influences additive genetic variance:</strong></p>
<p>Let’s define <span class="math inline">\(\rho_z\)</span> and <span class="math inline">\(\rho_g\)</span> to be the phenotypic and genetic correlations between mates and assume the regression of phenotypes of mates is linear. We also suppose that there are <span class="math inline">\(n\)</span> loci, each contributing equally to the genetic variance of the trait and define the parameter <span class="math inline">\(\gamma = 1 - [1/(2n)]\)</span>. With a random-mating base population with no LD and additive genetic variance <span class="math inline">\(\sigma^2_{A}\)</span>, a single generation of assortative mating will shift the additive genetic variance to:</p>
<p><span class="math display" id="eq:additive-genvar-onegen-AM">\[\begin{equation}
    \sigma^2_A(1) = (1 + \frac{\rho_z h^2} {2}) \sigma^2_A
    \tag{7.17}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(h^2 = \frac{\sigma^2_A} {\sigma^2_z}\)</span>. With continued assortative mating the variance asymptotically approaches the equilibrium:</p>
<p><span class="math display" id="eq:additive-genvar-onegen-AM">\[\begin{equation}
    \hat{\sigma}^2_A = \frac{\sigma^2_A} {1 - \gamma\hat{\rho_g}}
    \tag{7.17}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(\hat{\rho_g} = \rho_{z}\hat{\sigma}^2_A/\hat{\sigma}^2_z\)</span> is the equilibrium genetic correlation between mates. This can be re-written as:</p>
<p><span class="math display">\[\begin{equation}
    TO-DO!!!
\end{equation}\]</span></p>
<p>which gives the inflation of the additive equilibrium genetic variance relative to that in the random-mating base population (RECREATE FIGURE 7.7!).</p>
<p>The difference <span class="math inline">\(\hat{\sigma}^2_A - \sigma^2_A\)</span> is the additive disequilibrium covariance, _{A,A}, maintained by assortative mating. (UNCLEAR AS TO WHY!)</p>
<p>You can see from the figure, assortative mating must be fairly strong (<span class="math inline">\(\hat{\rho_g} \ge 0.2\)</span>) and combined with high <span class="math inline">\(h^2\)</span> to induce much change in the variance. Also, the effective number of loci has a negligible effect unless it is very small.</p>
<p>When only two loci contribute to the trait, <span class="math inline">\(\sigma^2_D\)</span> can change with assortative mating, but if the number of loci is even “moderate” the effect will be negligible so can be ignored for all complex human phenotypes for sure!</p>
<p>In addition to creating LD, assortative mating causes genotype frequencies for the selected trait to deviate from the Hardy-Weinberg expectations. Positive assortative mating leads to an increase in homozygosity and negative assortative mating has the opposite effect. However, <span class="math inline">\(\rho_z\)</span> and <span class="math inline">\(h^2\)</span> need to be very large to cause any major deviations from HWE. This implies the vast majority of genetic variance change induced by assortative mating comes in the form of increased LD, not changes in HWE, i.e. allelic association within rather than between gametes.</p>
<p>Table 7.4 in the book gives changes in additive and dominance components of covariance between relatives for an equilibrium population undergoing assortative mating.</p>
</div>
<div id="polyploidy-1" class="section level2" number="7.5">
<h2><span class="header-section-number">7.5</span> Polyploidy</h2>
<p>BORWANG!!! Ignoring!</p>
</div>
<div id="environmental-sources-of-covariance-between-relatives" class="section level2" number="7.6">
<h2><span class="header-section-number">7.6</span> Environmental sources of covariance between relatives</h2>
<p>This section essentially uses logic derived from path analysis to try and derive expressions for the environmental sources of covariance between various relatives including twins, full sibs and half sibs. To best follow the logic a deeper a look at path analysis and more time on the maths is likely required. COME BACK TO THIS!</p>
</div>
<div id="the-heritability-concept" class="section level2" number="7.7">
<h2><span class="header-section-number">7.7</span> The heritability concept</h2>
<p>Recalling that the first term in any genetic covariance expression is <span class="math inline">\(2\Theta_{xy}\sigma^2_{A}\)</span>. Thus, under the assumption that additive genetic variance is the dominant source of phenotypic covariance,</p>
<p><span class="math display" id="eq:approximate-narrow-sense-heritability">\[\begin{equation}
    h^2 = \frac{\sigma(z_x, z_y)} {2\Theta_{xy} \sigma^2_z}
    \tag{7.18}
\end{equation}\]</span></p>
<p>should provide a good approximation to the heritability. Violations of assumptions usually cause <span class="math inline">\(\sigma(z_x, z_y) / 2\Theta_{xy}\)</span> to be an upwardly biased estimator of <span class="math inline">\(\sigma^2_A\)</span>. A simple means of evaluating the likelihood of bias in heritability estimates arises when estimates of the phenotypic covariance are available for more than one type of relative (the book provides a couple of examples). The book also discusses lab vs. natural population estimates of heritability and gives examples. Conclusion: heritabilities respond to environmental change, and that substantial care should be taken in extrapolating results beyond the environment in which they are obtained!</p>
<p>EVERY TRAIT IS HERITABLE INIT! (pretty much)</p>
<p>In general, morphological characteristics tend to have higher heritabilities than life-history traits, with behavioural and physiological traits having intermediate heritabilites. Although there are plenty of exceptions of course.</p>
<div id="evolvability" class="section level3" number="7.7.1">
<h3><span class="header-section-number">7.7.1</span> Evolvability</h3>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="sources-of-environmental-variation.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="questions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["quant_genetics_book_notes.pdf", "quant_genetics_book_notes.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
