<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Sources of genetic variation for multilocus traits | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits</title>
  <meta name="description" content="Chapter 5 Sources of genetic variation for multilocus traits | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  <meta name="generator" content="bookdown 0.18.1 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Sources of genetic variation for multilocus traits | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Sources of genetic variation for multilocus traits | Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits" />
  
  
  

<meta name="author" content="Thomas Battram" />


<meta name="date" content="2020-04-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="properties-of-single-loci.html"/>

<script src="libs/header-attrs-2.1.1/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="an-overview-of-quantitative-genetics.html"><a href="an-overview-of-quantitative-genetics.html"><i class="fa fa-check"></i><b>1</b> An overview of quantitative genetics</a></li>
<li class="chapter" data-level="2" data-path="properties-of-distributions.html"><a href="properties-of-distributions.html"><i class="fa fa-check"></i><b>2</b> Properties of distributions</a></li>
<li class="chapter" data-level="3" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html"><i class="fa fa-check"></i><b>3</b> Covariance, regression, and correlation</a>
<ul>
<li class="chapter" data-level="3.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#covariance"><i class="fa fa-check"></i><b>3.1</b> Covariance</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#useful-identities-for-covariance"><i class="fa fa-check"></i><b>3.1.1</b> Useful identities for covariance</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#least-squares-linear-regression"><i class="fa fa-check"></i><b>3.2</b> Least squares linear regression</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#properties-of-least-squares"><i class="fa fa-check"></i><b>3.2.1</b> Properties of least squares</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#correlation"><i class="fa fa-check"></i><b>3.3</b> Correlation</a></li>
<li class="chapter" data-level="3.4" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#differential-selection-brief-intro"><i class="fa fa-check"></i><b>3.4</b> Differential selection (brief intro)</a></li>
<li class="chapter" data-level="3.5" data-path="covariance-regression-and-correlation.html"><a href="covariance-regression-and-correlation.html#correlation-between-genotype-and-phenotype-brief-intro"><i class="fa fa-check"></i><b>3.5</b> Correlation between genotype and phenotype (brief intro)</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html"><i class="fa fa-check"></i><b>4</b> Properties of single loci</a>
<ul>
<li class="chapter" data-level="4.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#introduction"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#allele-and-genotype-frequencies"><i class="fa fa-check"></i><b>4.2</b> Allele and genotype frequencies</a></li>
<li class="chapter" data-level="4.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-transmission-of-genetic-information"><i class="fa fa-check"></i><b>4.3</b> The transmission of genetic information</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-hardy-weinberg-principle"><i class="fa fa-check"></i><b>4.3.1</b> The Hardy-Weinberg principle</a></li>
<li class="chapter" data-level="4.3.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#sex-linked-loci"><i class="fa fa-check"></i><b>4.3.2</b> Sex-linked loci</a></li>
<li class="chapter" data-level="4.3.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#polyploidy"><i class="fa fa-check"></i><b>4.3.3</b> Polyploidy</a></li>
<li class="chapter" data-level="4.3.4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#age-structure"><i class="fa fa-check"></i><b>4.3.4</b> Age structure</a></li>
<li class="chapter" data-level="4.3.5" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#testing-for-hardy-weinberg-proportions"><i class="fa fa-check"></i><b>4.3.5</b> Testing for Hardy-Weinberg proportions</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#characterising-the-influence-of-a-locus-on-the-phenotype"><i class="fa fa-check"></i><b>4.4</b> Characterising the influence of a locus on the phenotype</a></li>
<li class="chapter" data-level="4.5" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#the-basis-of-dominance"><i class="fa fa-check"></i><b>4.5</b> The basis of dominance</a></li>
<li class="chapter" data-level="4.6" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#fishers-decomposition-of-the-genotypic-value"><i class="fa fa-check"></i><b>4.6</b> Fisher’s decomposition of the genotypic value</a></li>
<li class="chapter" data-level="4.7" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#partioning-the-genetic-variance."><i class="fa fa-check"></i><b>4.7</b> Partioning the genetic variance.</a></li>
<li class="chapter" data-level="4.8" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-effects-average-excesses-and-breeding-values"><i class="fa fa-check"></i><b>4.8</b> Additive effects, average excesses and breeding values</a></li>
<li class="chapter" data-level="4.9" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#extensions-for-multiple-alleles-and-non-random-mating"><i class="fa fa-check"></i><b>4.9</b> Extensions for multiple alleles and non random mating</a>
<ul>
<li class="chapter" data-level="4.9.1" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#average-excess"><i class="fa fa-check"></i><b>4.9.1</b> Average excess</a></li>
<li class="chapter" data-level="4.9.2" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-effects"><i class="fa fa-check"></i><b>4.9.2</b> Additive effects</a></li>
<li class="chapter" data-level="4.9.3" data-path="properties-of-single-loci.html"><a href="properties-of-single-loci.html#additive-genetic-variance"><i class="fa fa-check"></i><b>4.9.3</b> Additive genetic variance</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html"><i class="fa fa-check"></i><b>5</b> Sources of genetic variation for multilocus traits</a>
<ul>
<li class="chapter" data-level="5.1" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#epistasis"><i class="fa fa-check"></i><b>5.1</b> Epistasis</a></li>
<li class="chapter" data-level="5.2" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#a-general-least-squares-model-for-genetic-effects"><i class="fa fa-check"></i><b>5.2</b> A general least-squares model for genetic effects</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="sources-of-genetic-variation-for-multilocus-traits.html"><a href="sources-of-genetic-variation-for-multilocus-traits.html#extension-to-haploids-and-polyploids"><i class="fa fa-check"></i><b>5.2.1</b> Extension to haploids and polyploids</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Notes for: Walsh and Lynch. Genetics and Analysis of Quantitative Traits</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="sources-of-genetic-variation-for-multilocus-traits" class="section level1" number="5">
<h1><span class="header-section-number">Chapter 5</span> Sources of genetic variation for multilocus traits</h1>
<div id="epistasis" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Epistasis</h2>
<p>Epistasis describes the nonadditivity of effects between loci, i.e. the alleles of one loci influence the effects of another loci.</p>
<p>The genotypic value, <span class="math inline">\(G_{ijkl}\)</span>, needs to take into account all the interaction terms that can arrive between loci, for two loci it’s additive x additive effects (<span class="math inline">\(\alpha\alpha\)</span>), additive x dominance effects (<span class="math inline">\(\alpha\delta\)</span>), and dominance x dominance effects (<span class="math inline">\(\delta\delta\)</span>). As the number of loci increases the number of interaction terms increase steadily e.g. <span class="math inline">\(\alpha\alpha\alpha\)</span> will be there for three loci.</p>
</div>
<div id="a-general-least-squares-model-for-genetic-effects" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> A general least-squares model for genetic effects</h2>
<p>This is just an extension of the one-locus linear model introduced in Chapter 4.</p>
<p>For this section, imagine we are interested in measuring the genetic effects of two loci, <span class="math inline">\(G_{ijkl}\)</span>, which can easily be extended to more. The additive effect of an allele on a phenotype is just the phenotypic value in people with that allele minus the mean phenotypic value of the population. When considering epistatic effects we can define it in the same way.</p>
<p><span class="math display" id="eq:additive-effects-i">\[\begin{equation}
  \alpha_{i} = G_{i...} - \mu_{G}
  \tag{5.1}
\end{equation}\]</span></p>
<p><span class="math inline">\(G_{i...}\)</span> is just the conditional mean phenotype of individuals with allele <span class="math inline">\(i\)</span> at the first locus without regard to the other allele at that locus or to the genotype at the second locus. The other additive terms (for <span class="math inline">\(\alpha_{j}\)</span>, <span class="math inline">\(\alpha_{k}\)</span>, <span class="math inline">\(\alpha_{l}\)</span>) are defined in the same way. Within each locus, the mean value of average effects (weighted by allele frequency) = 0.</p>
<p>Dominance effects can be defined in a similar way, complete these equations by recalling the equation for <span class="math inline">\(G_{ij}\)</span> at the end of Chapter 4:</p>
<p><span class="math display" id="eq:dom-effects-lj" id="eq:dom-effects-ij">\[\begin{align}
  \delta_{ij} &amp;= G_{ij} - ... \tag{5.2} \\
  \delta_{lk} &amp;= G_{lk} - ... \tag{5.3}
\end{align}\]</span></p>
<p>Like with the additive effects, the mean dominance deviation at each locus is equal to zero.</p>
<p>Epistatic effect terms proceed in a similar fashion. Letting <span class="math inline">\(G_{i.k.}\)</span> be the mean phenotype of individuals with gene <span class="math inline">\(i\)</span> at locus 1 and <span class="math inline">\(k\)</span> at locus 2, without regard to the other two genes, the <span class="math inline">\(ik\)</span>th additive x additive effect is:</p>
<p><span class="math display" id="eq:add-add-effects-ik">\[\begin{equation}
  \left(\alpha\alpha\right)_{ik} = G_{i.k.} - \mu_{G} - \alpha{i} - \alpha{k} 
  \tag{5.4}
\end{equation}\]</span></p>
<p>So <span class="math inline">\(\left(\alpha\alpha\right)_{ik}\)</span> is the deviation of the conditional mean <span class="math inline">\(G_{i.k.}\)</span> from the expectation based on the population mean <span class="math inline">\(\mu_{G}\)</span> and the additive effects <span class="math inline">\(\alpha_{i}\)</span> and <span class="math inline">\(\alpha_{k}\)</span>. An additive x dominance effect measures the interaction between an allele at one locus with a genotype of another locus (see equation 5.5 in book) and the dominance x dominance effect involves an interaction between the genotypes at each locus (see equation 5.6 in book).</p>
<p>The complete genotypic value, <span class="math inline">\(G_{ijkl...}\)</span> can be found in equation 5.7 in the book. These parameters depend on genotype frequencies in the population, but the mean value of each type of effect is always equal to zero.</p>
<p>The genotypic value of an individual is often impossible to quantify because of variation in the phenotype due to the environment, but the genotypic value for an individual equation can be extended to populations. Providing mating is random and segregation of loci is independent, there is no statistical relationship between the genes found within or among loci. So the total genetic variance is just the sum of the variance of the individual effects, simplified this is:</p>
<p><span class="math display" id="eq:multilocus-genetic-variance">\[\begin{equation}
  \sigma^{2}_{G} = \sigma^{2}_{A} + \sigma^{2}_{D} + \sigma^{2}_{AA} + \sigma^{2}_{AD} + \sigma^{2}_{DD} + ...
  \tag{5.5}
\end{equation}\]</span></p>
<p>… here and in other cases just symbolises more terms can be added if more than two loci are used.</p>
<p>Epistatic effects are expected to be common throughout the genome and Wright thought they were the rule, rather than exception. See example two in the book for calculations of epistatic effects and how much variance they contribute to the overall genetic variance component. Overall, it is clear that even with large epistatic effects, additive genetic variance, <span class="math inline">\(\sigma^{2}_{A}\)</span> will pretty much always (if not always) contribute to the vast majority of overall genetic variance <span class="math inline">\(\sigma^{2}_{G}\)</span>. This is important for two reasons:</p>
<ol style="list-style-type: decimal">
<li>Variance components provide limited insight into the physiological mode of gene action, i.e. just because genetic variance is explained by additive effects (which means you essentially count each gene separately), it does not mean the interaction between genes is not important in terms of their function!</li>
<li>When interested in the variance of a trait that is explained by genetics, you can expect the vast majority of that variance to be explained by additive genetic effects, which makes things like estimating heritability far easier.</li>
</ol>
<div id="extension-to-haploids-and-polyploids" class="section level3" number="5.2.1">
<h3><span class="header-section-number">5.2.1</span> Extension to haploids and polyploids</h3>
<p>Skipped this section as not relevant to humans.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="properties-of-single-loci.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["quant_genetics_book_notes.pdf", "quant_genetics_book_notes.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
